diff --git a/guicast/bcresources.C b/guicast/bcresources.C
index c99ea22..6dc5d6c 100644
--- a/guicast/bcresources.C
+++ b/guicast/bcresources.C
@@ -78,11 +78,11 @@ const char* BC_Resources::medium_fontset = "-*-helvetica-bold-r-normal-*-14-*";
 const char* BC_Resources::large_fontset = "-*-helvetica-bold-r-normal-*-18-*";
 
 const char* BC_Resources::small_font_xft = N_("Sans:pixelsize=10.6667");
-const char* BC_Resources::small_font_xft2 = N_("-microsoft-verdana-*-*-*-*-*-*-*-*-*-*-*-*");
+const char* BC_Resources::small_font_xft2 = N_("-*-verdana-*-*-*-*-*-*-*-*-*-*-*-*");
 const char* BC_Resources::medium_font_xft = N_("Sans:pixelsize=13.3333");
-const char* BC_Resources::medium_font_xft2 = N_("-microsoft-verdana-*-*-*-*-*-*-*-*-*-*-*-*");
+const char* BC_Resources::medium_font_xft2 = N_("-*-verdana-*-*-*-*-*-*-*-*-*-*-*-*");
 const char* BC_Resources::large_font_xft = N_("Sans:pixelsize=21.3333");
-const char* BC_Resources::large_font_xft2 = N_("-microsoft-verdana-*-*-*-*-*-*-*-*-*-*-*-*");
+const char* BC_Resources::large_font_xft2 = N_("-*-verdana-*-*-*-*-*-*-*-*-*-*-*-*");
 
 suffix_to_type_t BC_Resources::suffix_to_type[] = 
 {
diff --git a/plugins/titler/title.C b/plugins/titler/title.C
index 923fde3..5dbad02 100644
--- a/plugins/titler/title.C
+++ b/plugins/titler/title.C
@@ -55,8 +55,7 @@
 
 #define ZERO (1.0 / 64.0)
 
-#define FONT_SEARCHPATH "fonts"
-//#define FONT_SEARCHPATH "/usr/X11R6/lib/X11/fonts"
+#define FONT_SEARCHPATH "/usr/share/fonts"
 
 
 REGISTER_PLUGIN(TitleMain)
@@ -1193,11 +1192,8 @@ void TitleMain::build_fonts()
 	if(!fonts)
 	{
 		fonts = new ArrayList<FontEntry*>;
-// Construct path from location of the plugin
 		char search_path[BCTEXTLEN];
-		strcpy(search_path, PluginClient::get_path());
-		char *ptr = strrchr(search_path, '/');
-		strcpy(ptr + 1, FONT_SEARCHPATH);
+		strcpy(search_path, FONT_SEARCHPATH);
 
 		FcPattern *pat;
 		FcFontSet *fs;
