diff --git a/configure.ac b/configure.ac
index b938520..0c73b36 100644
--- a/configure.ac
+++ b/configure.ac
@@ -785,7 +785,6 @@ AC_OUTPUT(Makefile cinelerra-cvs-current.spec po/Makefile.in \
 	plugins/timefront/Makefile \
 	plugins/svg/Makefile \
 	plugins/diffkey/Makefile \
-	plugins/fonts/Makefile \
 	image/Makefile)
 
 succeeded=yes
diff --git a/plugins/Makefile.am b/plugins/Makefile.am
index d1b6736..e927739 100644
--- a/plugins/Makefile.am
+++ b/plugins/Makefile.am
@@ -112,8 +112,7 @@ SUBDIRS= \
 	wipe \
 	yuv \
 	zoomblur \
-	$(MAYBE_FINDOBJECT) \
-	fonts
+	$(MAYBE_FINDOBJECT)
 
 AUTOMAKE_OPTIONS=foreign
 
